iothub/service/src/ServiceClient.cs

```diff
@@ -99,111 +100,75 @@ public static ServiceClient CreateFromConnectionString(string connectionString,
         public abstract FileNotificationReceiver<FileNotification> GetFileNotificationReceiver();
 
         /// <summary>
         /// Gets service statistics for the Iot Hub.
         /// </summary>
-        /// <returns>returns ServiceStatistics object containing current service statistics</returns>
-        public abstract Task<ServiceStatistics> GetServiceStatisticsAsync();
-
-        /// <summary>
-        /// Gets service statistics for the Iot Hub.
-        /// </summary>
-        /// <param name="cancellationToken">
-        /// The token which allows the the operation to be cancelled.
-        /// </param>
-        /// <returns>returns ServiceStatistics object containing current service statistics</returns>
-        public abstract Task<ServiceStatistics> GetServiceStatisticsAsync(CancellationToken cancellationToken);
-
-        /// <summary>
-        /// Interactively invokes a method on device
-        /// </summary>
-        /// <param name="deviceId">Device Id</param>
-        /// <param name="cloudToDeviceMethod">Device method parameters (passthrough to device)</param>
-        /// <returns>Method result</returns>
-        public abstract Task<CloudToDeviceMethodResult> InvokeDeviceMethodAsync(string deviceId, CloudToDeviceMethod cloudToDeviceMethod);
-
-        /// <summary>
-        /// Interactively invokes a method on device
-        /// </summary>
-        /// <param name="deviceId">Device Id</param>
-        /// <param name="cloudToDeviceMethod">Device method parameters (passthrough to device)</param>
-        /// <param name="cancellationToken">Cancellation Token</param>
-        /// <returns>Method result</returns>
-        public abstract Task<CloudToDeviceMethodResult> InvokeDeviceMethodAsync(string deviceId, CloudToDeviceMethod cloudToDeviceMethod, CancellationToken cancellationToken);
+        /// <param name="cancellationToken">A cancellation token to cancel the operation.</param>
+        /// <returns>The service statistics that can be retrieved from IotHub, eg. the number of device connected to the hub.</returns>
+        public abstract Task<ServiceStatistics> GetServiceStatisticsAsync(CancellationToken cancellationToken = default);
 
         /// <summary>
-        /// Interactively invokes a method on device
+        /// Interactively invokes a method on a device.
         /// </summary>
-        /// <param name="deviceId">Device Id</param>
-        /// <param name="moduleId">Module Id</param>
-        /// <param name="cloudToDeviceMethod">Device method parameters (passthrough to device)</param>
-        /// <returns>Method result</returns>
-        public abstract Task<CloudToDeviceMethodResult> InvokeDeviceMethodAsync(string deviceId, string moduleId, CloudToDeviceMethod cloudToDeviceMethod);
+        /// <param name="deviceId">The device identifier for the target device.</param>
+        /// <param name="cloudToDeviceMethod">Device method parameters (passthrough to the device).</param>
```

Comment:
Wondering if we need to say - passthrough to the device. The class itself has a better description I felt - "Parameters to execute a direct method on the device"